libraries/AP_HAL_Linux/RCOutput_Bebop.cpp

```diff
@@ -283,6 +283,37 @@ void RCOutput_Bebop::_play_sound(uint8_t sound)
     _dev->get_semaphore()->give();
 }
 
+/*
+ * pwm is the pwm power used for the note.
+ *  It has to be >= 3, otherwise it refers to a predefined song
+ * (see _play_sound function)
+ * period is in us and duration in ms.
+ */
+void RCOutput_Bebop::play_note(uint8_t pwm, uint16_t period, uint16_t duration)
+{
+    struct PACKED {
+        uint8_t header;
+        uint8_t pwm;
+        be16_t period;
+        be16_t duration;
+    } msg;
+
+    if (pwm < 3)
+        return;
```

Comment:
coding style requires braces.... if it was only this problem I'd fix it myself while applying, but see the other comment.